lib/web_ui/dev/README.md

```diff
@@ -89,6 +89,28 @@ To run tests on Safari use the following command. It works on MacOS devices and
 felt test --browser=safari
 ```
 
+One can also use Ios Safari for running unit tests. It works on MacOS devices. There are few prerequisite steps:
```

Comment:
nit: iOS, macOS
nit: "iOS Safari...works on macOS devices" sounds confusing, because in production iOS Safari runs on iOS devices. Suggestion: "One can also use Safari running in iOS Simulator for running unit tests". You don't have to talk about macOS at all, because we already know that iOS Simulator requires macOS.

Reply:
done.